bears/python/requirements/PySafetyBear.py

```diff
@@ -0,0 +1,96 @@
+from collections import namedtuple
+import pkg_resources
+import re
+
+from safety import safety
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.bears.requirements.PipRequirement import PipRequirement
+from coalib.results.Result import Result
+from coalib.results.SourceRange import SourceRange
+from coalib.settings.Setting import typed_list
+
+
+# the safety module expects an object that looks like this
+# (not importing it from there because it's in a private-ish location)
+Package = namedtuple('Package', ('key', 'version'))
+
+
+class PySafetyBear(LocalBear):
+    """
+    Checks if any of your Python dependencies have known security issues.
+
+    Data is taken from pyup.io's vulnerability database hosted at
+    https://github.com/pyupio/safety.
+    """
+
+    LANGUAGES = {
+        'Python Requirements',
+        'Python 2 Requirements',
+        'Python 3 Requirements',
+    }
+    AUTHORS = {'Bence Nagy'}
+    REQUIREMENTS = {PipRequirement('safety', '0.3.*')}
+    AUTHORS_EMAILS = {'bence@underyx.me'}
+    LICENSE = 'MIT'
+    CAN_DETECT = {'Security'}
+
+    def run(self, filename, file):
+        """
+        Checks for vulnerable package versions in requirements files.
+        """
+        packages = list(
+            Package(key=req.key, version=req.specs[0][1])
+            for req in self.try_parse_requirements(file)
+            if len(req.specs) == 1 and req.specs[0][0] == '=='
+        )
+
+        if not packages:
+            raise StopIteration
+
+        for vulnerability in safety.check(packages=packages):
+            if vulnerability.is_cve:
+                message_template = (
+                    '{vuln.name}{vuln.spec} is vulnerable to {vuln.cve_id} '
+                    'and your project is using {vuln.version}.'
+                )
+            else:
+                message_template = (
+                    '{vuln.name}{vuln.spec} is vulnerable and your project is '
+                    'using {vuln.version}.'
+                )
+
+            # StopIteration should not ever happen so skipping its branch
+            line_number, line = next(  # pragma: no branch
+                (index, line) for index, line in enumerate(file, start=1)
+                if vulnerability.name in line
+            )
```

Comment:
wouldn't it e simpler to check the packages one by one? Then you immediately know which line that's about